backend/src/models/IPost.ts

```diff
@@ -0,0 +1,16 @@
+export interface IPost {
```

Comment:
Do not prefix interfaces with 'I':
 https://github.com/microsoft/TypeScript-Handbook/issues/121